### PR TITLE
Add company info for Carbon Black 

### PR DESCRIPTION
**Pull request by hanapotski**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [x] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @hanapotski, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 01ff398).  See below for details:

```
carbon-black.md: Company title "Bairesdev" doesn't match filename (expected ~ "bairesdev.md")

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

File: company-profiles/carbon-black.md [hunk 1]
```diff
@@ -1,11 +1,85 @@
-# Carbon Black
+# Bairesdev
```
Comment (dougaitken) [inline company-profiles/carbon-black.md]: Can you change this, please?

Comment (hanapotski) [reply company-profiles/carbon-black.md]: updated, sorry about that ✌️

**Comment by auto-nylen** [comment]
Hi @hanapotski, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 97daf65).  See below for details:

```
bairesdev.md: Company title "Carbon Black" doesn't match filename (expected ~ "carbon-black.md")
carbon-black.md: Company title "Bairesdev" doesn't match filename (expected ~ "bairesdev.md")

2 problems detected
```

Please fix these issues and push a new commit to this pull request.

File: company-profiles/carbon-black.md (hunk 1 above)
Comment (nylen) [reply company-profiles/carbon-black.md]: Looks like you updated the wrong file there.

File: company-profiles/bairesdev.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Bairesdev
```
Comment (dougaitken) [inline company-profiles/bairesdev.md]: Can this be changed back to Bairesdev

File: company-profiles/carbon-black.md (hunk 1 above)
Comment (dougaitken) [reply company-profiles/carbon-black.md]: Can this one be changed back to Carbon Black, please?

**Comment by auto-nylen** [comment]
Hi @dougaitken, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 01d3e75).  See below for details:

```
carbon-black.md: Company title "Bairesdev" doesn't match filename (expected ~ "bairesdev.md")

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @dougaitken, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 6b3b6fc). :heavy_check_mark:

**Pull request merged**